src/ViewArticle/tests/ViewArticlereducer.test.jsx

```diff
@@ -0,0 +1,141 @@
+import ViewArticleReducer from '../redux/reducers/ViewArticlereducer';
+import ArticleTypes from '../redux/types';
+
+describe('test for viewArticle reducer', () => {
+  it('Testing get single article success', () => {
+    const article = {
+      title: 'Logged in',
+      description: 'You can close this page and return to your CLI. It should now be logged in.',
+      body: 'You can close this page and return to your CLI. It should now be logged in.',
+      createdAt: '2019-07-22T05:19:22.188734Z',
+      updatedAt: '2019-07-22T05:19:22.188788Z',
+      author: {
+        email: 'alexxsanya@gmail.com',
+        username: 'alexxsanya',
+        bio: '',
+        image: '',
+        gender: 'male',
+        first_name: 'Alex',
```

Comment:
@jennizalwango Please do note that the naming convention you are using here is for python and not javascript, that convention can only be acceptable for 'ACTIONS', I kindly request you review this and make the changes with the camelcase naming convention. And in this case `first_name` ought to be `firstName` nonetheless excellent work.

Reply:
Kindly note that here l am getting data as it is in the backend l am not creating the article there but l am just picking it as it is in the backend , for more infor you could look at swagger and see how data is and its naming conventions